.ci/magic-modules/create-pr.sh

```diff
@@ -149,4 +149,8 @@ I am (still) a robot that works on MagicModules PRs!
 I just wanted to let you know that your changes (as of commit $(git rev-parse --short HEAD~1)) have been included in your existing downstream PRs.
 EOF
 
+  # Create blank label file
```

Comment:
Can you answer why we need a blank label file in the comment? Also, will `$LABELS` always be blank here?

Reply:
I put in an explanation for why we need a blank label file (you're right, $LABELS will always be blank in that case).

I believe that situation is only used in situations where there are no changes to the MM codebase between the last Magician run, but I'm not entirely sure. @ndmckinley, any insight?

Reply:
(just saw first note)

New labels should get added on successive runs, but there's some of this Bash script that I don't fully understand. @ndmckinley would have better insight than I do.

Reply:
Let's go through it together - come find me whenever.  :)

Reply:
We went through this, this is right.  I would probably just `echo "" > ./label_file` instead - a little simpler, but either way is fine.